search/api.py

```diff
@@ -1,306 +1,90 @@
-"""
```

Comment:
probably best to view this on its own. its contents were moved to search/indexing_api.py. the contents here are new and unrelated